website/content/docs/oss/configuration/worker/self-managed-worker.mdx

```diff
@@ -0,0 +1,56 @@
+---
+layout: docs
+page_title: Self-Managed Worker Configuration
+description: |-
+  Self-Managed worker-specific parameters.
+---
+
+
+## Self-Managed Worker Configuration
+Self-Managed Workers authenticate to Boundary using a new certificate-based method, allowing for worker deployment
+without using a shared KMS.
+
+Self-Managed Workers require an accessible directory, defined by `auth_storage_path`, for credential storage.
+Example (not safe for production!):
+
+```hcl
+worker {
+  auth_storage_path="/tmp/boundary/demo-worker-1"
+  initial_upstreams = ["10.0.0.1"]
+}
+```
+
+__Note__: `name` and `description` fields are not valid for Self-Managed workers. These fields are only valid for
```

Comment:
Might be good to make it clear that they can still be set, just through the API -- they're not valid in the config for these workers.

Reply:
Updated to:
__Note__: `name` and `description` fields are not valid config fields for Self-Managed workers. These fields are only valid for [KMS Workers][]. `name` and `description` can only be set for Self-Managed workers through the API.